packages/dd-trace/test/appsec/blocking.spec.js

```diff
@@ -1,40 +1,36 @@
 'use strict'
 
+const { expect } = require('chai')
```

Comment:
are you sure we need expect here ? it's supposed to be injected automatically by the test setup no ?